readiness/controller/pkg/readinessprovider/suite_test.go

```diff
@@ -114,6 +182,13 @@ var _ = AfterSuite(func() {
 })
 
 var _ = Describe("Readiness Provider controller", func() {
```

Comment:
Can we add tests for Update and Delete as well.

Reply:
Done